autosubmit/monitor/monitor.py

```diff
@@ -625,7 +630,7 @@ def generate_output_stats(self, expid: str, joblist: list[Job], output_format="p
         output_complete_path_stats = os.path.join(BasicConfig.DEFAULT_OUTPUT_DIR, output_filename)
         is_default_path = True
         if is_owner or is_eadmin:
-            create_path_if_not_exists_group_permission(os.path.join(BasicConfig.LOCAL_ROOT_DIR, expid, "stats"))
+            Path(Path(BasicConfig.LOCAL_ROOT_DIR, expid, "stats")).mkdir(exist_ok=True, mode=0o774)
```

Comment:
s/mode=0o774/mode=_DEFAULT_MKDIR_GROUP_PERMISSION (and for my OCD the order could match the one below :rofl: )